packages/flutter/lib/src/foundation/internal_web.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The largest SMI value.
+///
+/// See <https://www.dartlang.org/articles/numeric-computation/#smis-and-mints>
```

Comment:
Should this get the same doc comment as `internal.dart`? It strange that the comment is `The largest SMI value`, but the value is not that.

Could we use dartdoc templates here to make it very clear for people reading the code that this should be documented with the same comment?